client/src/components/IngredientList.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types'
+import React from 'react';
+
+const IngredientList =  (props) => {
+  let ingredientsArray = props.ingredients.split(",");
```

Comment:
Great work, however, use const instead of let for variables that won't be reassigned

Reply:
Thanks for your feedback, will review and make changes